app/models/effective/form_input.rb

```diff
@@ -121,17 +121,15 @@ def build_input_group_content(&block)
       if layout == :horizontal
         build_input_group { build_input(&block) } + build_hint
       else
-        build_label + build_input_group { build_input(&block) } + build_hint
+        build_label + build_input_group { build_input(&block) } + build_feedback + build_hint
       end
     end
 
     def build_input_group(&block) # Includes input and feedback
       content_tag(:div, '', options[:input_group][:input_group]) do # Twice here, kind of weird.
         [
           (content_tag(:div, options[:input_group][:prepend], class: 'input-group-prepend') if options[:input_group][:prepend]),
-          build_input(&block),
-          (content_tag(:div, options[:input_group][:append], class: 'input-group-append') if options[:input_group][:append]),
```

Comment:
I think this input group append stuff needs to be restored

Reply:
feedback being within the input group is specifically the bug/error that i'm trying to fix. here it is in the bs4 documentation where the feedback it outside of the feedback group: https://getbootstrap.com/docs/4.6/components/forms/#supported-elements

Reply:
I see that the HTML listed there is different than my reference HTML on

https://getbootstrap.com/docs/4.0/components/forms/#server-side

So yea, I dunno lol

Reply:
wrong version, you linked 4.0 and i linked 4.6